src/sessionTypes/auth0WebAuth.js

```diff
@@ -193,24 +209,24 @@ class Auth0WebAuth {
   }
 
   /**
-   * Logs the user out by removing any stored session info, clearing any token renewal, and
-   * redirecting to the root
+   * Logs the user out by removing any stored session info, clearing any token
+   * renewal, and redirecting to the root
    */
   logOut() {
     this._sessionInfo = {};
+    this._tokenPromises = {};
 
     localStorage.removeItem('access_token');
-    localStorage.removeItem('api_token');
     localStorage.removeItem('expires_at');
 
-    clearTimeout(this._tokenRenewalTimeout);
+    clearTimeout(this._sessionRenewalTimeout);
 
-    this._onRedirect('/');
+    this._auth0.logout({ returnTo: new URL(window.location).origin });
```

Comment:
This isn't necessarily an issue but `URL` isn't supported in any version of IE. Given that the SDK could be used by some third party, do we want to try and support IE? Feel free to tell me to go away.

Reply:
`import URL from 'url-parse';` is at the top of the file -- it's actually this package with IE support down to IE9: https://github.com/unshiftio/url-parse#readme

Reply:
Carry on sir

Reply:
Sorry, I see it now... It wasn't in the diff because it was already in the file 🤦‍♂